roles/config-rh-sso/templates/realm.json.j2

```diff
@@ -0,0 +1,14 @@
+{
+  "id": "{{ realm_id | default(realm_name) }}",
+  "realm": "{{ realm_name }}",
+  "displayName": "{{ realm_displayName }}",
+  "enabled": "{{ realm_enabled | default(true) }}",
+  "sslRequired": "{{ realm_sslRequired | default('something-that-makes-sense') }}",
```

Comment:
`something-that-makes-sense`? Doesn't look like a proper default value(?)

Reply:
Fixed.